modern/src/settings/PreferencesPage.jsx

```diff
@@ -272,8 +272,8 @@ const PreferencesPage = () => {
                   label={t('devicePrimaryInfo')}
                 />
                 <SelectField
-                  emptyValue=""
-                  value={attributes.deviceSecondary || ''}
+                  emptyValue={null}
```

Comment:
Shouldn't `null` be the default already?

Reply:
Yeah generally speaking I think we can change the default value from "" to `null`, will double check if there are any differences between `Select` and `Autocomplete` in this regard - but I mostly changed it to `null` as otherwise `Autocomplete` generated a Warning (but still worked without a hitch).

Reply:
I meant that we don't need to explicitly set it to null in every place.

Reply:
So, emptyValue difference between `0` and `null` apparently stems from Issue #802 ? And there is difference on how this is set in the old interface?

Reply:
But I guess we can default to `null` as now most of initial values (if no value stored) are defaulting to `null` instead of 0.

Reply:
We are now defaulting to `null` and it works ok.